### ISSUE TITLE
How do I save my fine-tune checkpoint?

### ISSUE THREAD

**Issue by znqaq** [PriorLabs/TabPFN#292]
1. Thank you for your work, I have achieved good results. 
But I don't know how to save the model weights like `torch.save()`,.
I know that tablePFN provides the api 'model_path' for load ckpt, so I want to take advantage of this to load my fine-tine model.

```
model = TabPFNRegressor()
model.fit(X,Y)
 
**how to save ckpt ?**
```

2. Your code provides a 'device' interface to use the GPU for training, but it seems like this method can't support training on multiple GPUs at the same time.

**Comment by iivalchev** [comment]
No need to save it. TabPFN does in-context learning (ICL), so fitting and inference happens during the forward pass (`predict*` method). Model weights are not affected. From your example what will happen during `fit` is that the preprocessing pipeline will be initialized and fitted. As far as I know you can't persist that.

**Comment by iivalchev** [comment]
Regarding the mutli-GPU support. You can try wrapping the transformer in `DataParallel` after `fit`.

```
from torch.nn import DataParallel

model = TabPFNRegressor()
model.fit(X,Y)
model.model_ = DataParallel(model.model_)
model.executor_.model = model.model_
```

**Comment by znqaq** [comment]
Alright, I get it, I've learned a lot, thanks!

**Comment by znqaq** [comment]
I didn't have a sufficient understanding of ICL before, sorry.
